code/modules/surgery/organs/external/tails.dm

```diff
@@ -32,6 +34,42 @@
 		else if(type in receiver.dna.species.external_organs)
 			receiver.add_mood_event("wrong_tail_regained", /datum/mood_event/tail_regained_wrong)
 
+/obj/item/organ/external/tail/on_bodypart_insert(obj/item/bodypart/bodypart)
+	var/obj/item/organ/external/spines/our_spines = bodypart.owner.get_organ_slot(ORGAN_SLOT_EXTERNAL_SPINES)
+	if(!isnull(our_spines))
```

Comment:
```suggestion
	if(our_spines)
```
!isnull is for converting to a boolean (for like return values or something), which is already done implicitly by if-statements